velox/dwio/catalog/fbhive/test/FileUtilsTests.cpp

```diff
@@ -31,6 +33,7 @@ TEST(FileUtilsTests, makePartName) {
   ASSERT_EQ(
       FileUtils::makePartName(pairs, false),
       "ds=2016-01-01/FOO=__HIVE_DEFAULT_PARTITION__/a%0Ab%3Ac=a%23b%3Dc");
+  ASSERT_THROW(FileUtils::makePartName({}, false), VeloxException);
```

Comment:
Thank you. Would be nice to also update documentation for this API to clarify this behavior.